blazor/pdfviewer-2/server-deployment/deploy-maui-android.md

```diff
@@ -0,0 +1,190 @@
+
+* Add the script in the `<head>` of the **~/wwwroot/index.html** file.
+
+{% tabs %}
+{% highlight html %}
+
+<head>
+    <!-- Syncfusion Blazor SfPdfViewer controls theme style sheet -->
+    <link href="_content/Syncfusion.Blazor.Themes/bootstrap5.css" rel="stylesheet" />
+    <!-- Syncfusion Blazor SfPdfViewer controls scripts -->
+    <script src="_content/Syncfusion.Blazor.SfPdfViewer/scripts/syncfusion-blazor-sfpdfviewer.min.js" type="text/javascript"></script>
+</head>
+
+{% endhighlight %}
+{% endtabs %}
+
+> Checkout [Adding Script Reference topic](https://blazor.syncfusion.com/documentation/common/adding-script-references) to learn different ways to add script reference in Blazor Application. 
+
+> Syncfusion recommends to reference scripts using [Static Web Assets](https://blazor.syncfusion.com/documentation/common/adding-script-references#static-web-assets), [CDN](https://blazor.syncfusion.com/documentation/common/adding-script-references#cdn-reference) and [CRG](https://blazor.syncfusion.com/documentation/common/custom-resource-generator) by [disabling JavaScript isolation](https://blazor.syncfusion.com/documentation/common/adding-script-references#disable-javascript-isolation) for better loading performance of the Blazor application.
+
+## Adding Blazor SfPdfViewer Component
+
+Add the Syncfusion SfPdfViewer component in the **~/Pages/Index.razor** file.
+
+{% tabs %}
+{% highlight razor %}
+
+@page "/"
+
+<SfPdfViewer2 @ref="viewer" DocumentPath="@DocumentPath" Height="100%" Width="100%"></SfPdfViewer2>
+
+@code {
+    SfPdfViewer2 viewer;
+    private string DocumentPath { get; set; } = "";
+
+    protected override void OnInitialized()
+    {
+        string basePath = "MauiBlazorAndroid.wwwroot.data.pdf_succinctly.pdf";
+
+        Stream DocumentStream = this.GetType().Assembly.GetManifestResourceStream(basePath);
+        DocumentStream.Position = 0;
+        using (MemoryStream memoryStream = new MemoryStream())
+        {
+            DocumentStream.CopyTo(memoryStream);
+            byte[] bytes = memoryStream.ToArray();
+            string base64String = Convert.ToBase64String(bytes);
+            string base64prefix = "data:application/pdf;base64,";
+            //Assigned the base64 path to the PDF document path.
+            DocumentPath = $"{base64prefix}{base64String}";
+        }
+        base.OnInitialized();
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+N> When developing a Blazor Android MAUI application, it is need to pass the `DocumentPath` to the SfPdfViewer component as a `base64 string`. This ensures that the application can retrieve and render the PDF document correctly within the SfPdfViewer.

```

Comment:
N> When developing a Blazor Android MAUI application, passing the `DocumentPath` to the SfPdfViewer component as a `base64 string` is needed. This ensures that the application can retrieve and render a PDF document correctly within the SfPdfViewer.